man/src/oxide-config.1.md

```diff
@@ -0,0 +1,196 @@
+% OXIDE-CONFIG(1) oxide-config 0.1.0
```

Comment:
Note: This file is definetily out of date. Please remove from this branch and only use the one on your branch doc/issue97-create-man-page-for-config-file .